fixtures/annotations.py.native

```diff
@@ -1,148 +1,140 @@
-                                    "lineno": 3
-                                },
-                                "arg": "a",
-                                "ast_type": "arg",
-                                "col_offset": 14,
-                                "end_col_offset": 14,
-                                "end_lineno": 3,
-                                "lineno": 3
-                            },
-                            {
-                                "annotation": {
-                                    "ast_type": "Name",
-                                    "col_offset": 25,
-                                    "ctx": "Load",
-                                    "end_col_offset": 27,
-                                    "end_lineno": 3,
-                                    "id": "str",
-                                    "lineno": 3
-                                },
-                                "arg": "b",
-                                "ast_type": "arg",
-                                "col_offset": 22,
-                                "end_col_offset": 22,
-                                "end_lineno": 3,
-                                "lineno": 3
-                            }
-                        ],
-                        "ast_type": "arguments",
-                        "defaults": [],
-                        "kw_defaults": [],
-                        "kwarg": null,
-                        "kwonlyargs": [],
-                        "vararg": null
-                    },
-                    "ast_type": "FunctionDef",
-                    "body": [
-                        {
-                            "ast_type": "Return",
-                            "col_offset": 5,
-                            "end_col_offset": 10,
-                            "end_lineno": 4,
-                            "lineno": 4,
-                            "value": {
-                                "ast_type": "Num",
-                                "col_offset": 12,
-                                "end_col_offset": 14,
-                                "end_lineno": 4,
-                                "lineno": 4,
-                                "n": 0
-                            }
-                        }
-                    ],
-                    "col_offset": 5,
-                    "decorator_list": [],
-                    "end_col_offset": 12,
-                    "end_lineno": 3,
-                    "lineno": 3,
-                    "name": "somefunc",
-                    "returns": {
-                        "ast_type": "Name",
-                        "col_offset": 33,
-                        "ctx": "Load",
-                        "end_col_offset": 37,
-                        "end_lineno": 3,
-                        "id": "float",
-                        "lineno": 3
-                    }
-                }
-            ]
-        }
-    }
-}
+   'PY3AST': {
```

Comment:
Please regenerate the fixtures as well

Reply:
Odd. I deleted the fixtures, updated the SDK, did `bblfsh-sdk update` on the driver dir and regenerated the fixtures with `go test` and is still using that format.